### ISSUE TITLE
Basemap- OSM not showing any result on display

### ISSUE THREAD

**Issue by vishnue1993** [domlysz/BlenderGIS#102]
Am trying to import OSM using BlenderGIS , but am able to see a white screen only after doing this. 
I have set the CRS as Web Mercator. 
![display_osm](https://user-images.githubusercontent.com/30351861/37387755-bfcbb214-2783-11e8-8610-7c5d38d23b75.png)

Please provide some suggestions.

**Comment by vishnue1993** [comment]
Still I am having same problem ,. I am able to access Bing, Google, Even OSM WMS also. 
But OSM is not accessible. while screen displayed. Doesnt matter zooming in and out.

**Comment by vishnue1993** [comment]
I started working in BlenderGIS.... Really great boss
I would like to know the possibility of creating pipes for the shapefile (pipeline as line feature shapefile to 3D pipes)/.
I would like to know about this
I was not able to zoom those shapefiles too much as they are disappearing. Please give some suggestions. Thanks inn advance

**Comment by PayamT** [comment]
Yes you can, import the shape file, convert it to Curve and bevel it using any profile you need. You can smooth it in the sharp bending angles. 


Sent from Yahoo Mail for iPhone


On Saturday, March 17, 2018, 1:22 PM, vishnue1993 <notifications@github.com> wrote:


I started working in BlenderGIS.... Really great boss
I would like to know the possibility of creating pipes for the shapefile (pipeline as line feature shapefile to 3D pipes)/.
I would like to know about this
I was not able to zoom those shapefiles too much as they are disappearing. Please give some suggestions. Thanks inn advance

**Comment by vishnue1993** [comment]
Yeah...i got it...it works...thanks
